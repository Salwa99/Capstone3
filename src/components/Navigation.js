import React from 'react';
import { NavLink } from 'react-router-dom';
import { TiMicrophone } from 'react-icons/ti';
import { IoMdSettings, IoIosArrowBack } from 'react-icons/io';
import house from '../assets/house.png';

function Navigation() {
  return (
    <div className="navbar">
      <ul>
        <li className="back">
          <NavLink to="/">
            <IoIosArrowBack />
          </NavLink>
        </li>
      </ul>
      <div className="logo">
        <div className="wrapper">
          <div className="bg"> Hogwarts characters </div>
          <div className="fg"> Hogwarts characters </div>
        </div>
        <img src={house} alt="hogwarts houses" />
      </div>
      <div>
        <div className="setting">
          <TiMicrophone />
          <IoMdSettings />
        </div>
      </div>
    </div>
  );
}

export default Navigation;
