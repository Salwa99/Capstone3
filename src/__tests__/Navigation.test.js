import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Navigation from '../components/Navigation';
import store from '../redux/ConfigueStore';

const App = () => {
  <Provider store={store}>
    <Navigation />
  </Provider>;
};

describe('Navigation component test', () => {
  it('Testing Navigation rendering', () => {
    const NavigationComponent = renderer.create(<App />).toJSON();
    expect(NavigationComponent).toMatchSnapshot();
  });
});
