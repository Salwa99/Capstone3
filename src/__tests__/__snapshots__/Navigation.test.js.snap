// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component test Testing Navigation rendering 1`] = `null`;
